pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[project]
+name = "rids"
+version = "0.1.0"
+description = "Remote Intrusion Detection System"
+readme = "README.md"
+requires-python = ">=3.7"
+dependencies = [
+  "absl-py",
+]
+
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tools.setuptools]
+packages = ["pyrids"]
```

Comment:
Don't you need to define that package in the source code?

Reply:
yes, that's part of the follow-up commit mentioned in the PR description -- it requires moving around files and a significant change to how install.sh works and I'd like to get the service calling `rids` directly too now that we don't need the shell script for any setup.  I figured I'd give you a chance to comment on the .toml file first, though.